01_basic/src/main/resources/basic.sw.yaml

```diff
@@ -20,7 +20,6 @@ functions:
     type: expression
     operation: '{
       "result": {
-      "completedWith":"success",
```

Comment:
let's split the the changes for the result to its own PR, so we can focus only on the demo for this PR

Reply:
see https://github.com/rhdhorchestrator/orchestrator-demo/pull/18